data/services/cdn-route.toml

```diff
@@ -1,3 +1,3 @@
 name = "cdn-route"
-description = "Custom routes with HTTPS certificates using Amazon CloudFront"
+description = "Custom domains, CDN caching, and TLS certificates using Amazon CloudFront"
```

Comment:
This should match whatever we change above for CDN routes. Should we have to update both manually, or is there a way in Hugo to use this data inside pages?